src/widgets/splits/SplitInput.cpp

```diff
@@ -303,13 +302,16 @@ void SplitInput::updateCancelReplyButton()
 {
     float scale = this->scale();
 
-    auto text =
-        QStringLiteral("<img src=':/buttons/%1.svg' width='%2' height='%2' />")
-            .arg(this->theme->isLightTheme() ? "cancelDark" : "cancel")
-            .arg(int(12 * scale));
-
-    this->ui_.cancelReplyButton->getLabel().setText(text);
-    this->ui_.cancelReplyButton->setFixedHeight(int(12 * scale));
+    if (this->theme->isLightTheme())
+    {
+        this->ui_.cancelReplyButton->setSvgResource(":/buttons/cancelDark.svg");
+    }
+    else
+    {
+        this->ui_.cancelReplyButton->setSvgResource(":/buttons/cancel.svg");
+    }
+    this->ui_.cancelReplyButton->setFixedHeight(int(24 * scale));
+    this->ui_.cancelReplyButton->setFixedWidth(int(24 * scale));
```

Comment:
We can retain the old size by disabling the margin for now

```suggestion
    this->ui_.cancelReplyButton->setEnableMargin(false);
    this->ui_.cancelReplyButton->setFixedHeight(int(12 * scale));
    this->ui_.cancelReplyButton->setFixedWidth(int(12 * scale));
```